### PR TITLE
Create jrtidev.json

### PR DESCRIPTION
**Pull request by jrtidev**

### CODE REVIEW

**Comment by droderiques** [comment]
looks good

**Comment by droderiques** [comment]
done

File: pins/jrtidev.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: jrtidev
```
Comment (crichID) [inline pins/jrtidev.json]: @jrtidev you will need to add three dashes `---` on the first and last line of the file for it to work properly.

**Comment by crichID** [comment]
@jrtidev I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3342/commits/a7c6f3e6e3cec730e32bc934b3dd1c3e5517ec27. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 10th, I will go ahead and do it just to keep the repository clean!

**Comment by ElizabethSorenson** [comment]
looks great

**Pull request merged**